### PR TITLE
Make agent doers' tocks configurable from habitat config file

### PR DESCRIPTION
**Pull request by rubelhassan**
#258

### CODE REVIEW

File: tests/app/test_agenting.py [hunk 1]
```diff
@@ -62,6 +64,52 @@ def test_load_ends(helpers):
...
+        assert agent.tocks == {
+            "initer": 0.0,
+            "escrower": 1.0
+        }
+
+        escrower_doer = next((doer for doer in agent.doers if isinstance(doer, agenting.Escrower)), None)
+        assert escrower_doer is not None
+        assert escrower_doer.tock == 1.0
+
+        querier_docer = next((doer for doer in agent.doers if isinstance(doer, agenting.Initer)), None)
```
Comment (2byrds) [inline tests/app/test_agenting.py:101]: did you intend the name 'docer' here and below? If so, what is the distinction?

Comment (rubelhassan) [reply tests/app/test_agenting.py:101]: It's a typo, I meant `doer`, I'll update it. Thanks.

**Comment by 2byrds** [comment]
@rubelhassan LGTM. if you were able to successfully run the signify-ts integration scripts against a keria with these changes I'll approve it.

File: src/keria/app/agenting.py [hunk 2]
```diff
@@ -289,6 +290,8 @@ def __init__(self, hby, rgy, agentHab, agency, caid, **opts):
         self.agentHab = agentHab
         self.agency = agency
         self.caid = caid
+        self.cfd = MappingProxyType(dict(self.hby.cf.get()) if self.hby.cf is not None else dict())
```
Comment (pfeairheller) [inline src/keria/app/agenting.py:293]: Why do we need MappingProxyType here instead of accessing the dict directly?

**Review by pfeairheller — CHANGES_REQUESTED** [review]

File: src/keria/app/agenting.py (hunk 2 above)
Comment (rubelhassan) [reply src/keria/app/agenting.py:293]: I've used it to make the `dict` immutable, so it cannot be changed unintentionally.

**Review by pfeairheller — APPROVED** [review]

**Comment by rubelhassan** [comment]
> I just resolved the conflicts with main for this PR. Once it builds I'll merge it.

Thank you, I was about update the PR :)

**Pull request merged**